app/controllers/graphql_controller.rb

```diff
@@ -1,12 +1,11 @@
+require_relative '../graphql/utilities/credential'
+
 class GraphqlController < ApplicationController
   def execute
     variables = ensure_hash(params[:variables])
     query = params[:query]
     operation_name = params[:operationName]
-    context = {
-      # Query context goes here, for example:
-      # current_user: current_user,
-    }
+    context = { current_user: current_user }
```

Comment:
Will this pass `{ current_user: nil }` as context to the `execute` method? If that's ok internally i guess that is fine, be careful passing missing values in hashes around as sometimes checks are run just on the key and can lead to nil values making there way into use.

FYI - you can call `{}.compact` to filter any missing values.

Reply:
I think context is only used when I explicitly ask for it, and I only do so when checking is current_user is truthy. But it may be wise to add compact for future proofing.